allennlp/tests/data/token_indexers/pretrained_transformer_indexer_test.py

```diff
@@ -3,24 +3,74 @@
 from allennlp.common.testing import AllenNlpTestCase
 from allennlp.data import Token, Vocabulary
 from allennlp.data.token_indexers import PretrainedTransformerIndexer
+from allennlp.data.tokenizers import PretrainedTransformerTokenizer
 
 
 class TestPretrainedTransformerIndexer(AllenNlpTestCase):
-    def test_as_array_produces_token_sequence(self):
+    def test_as_array_produces_token_sequence_bert(self):
         tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+        allennlp_tokenier = PretrainedTransformerTokenizer("bert-base-uncased")
+
+        # uncased
```

Comment:
A block level comment like this suggests that the block should be broken out into its own test case. (Especially since the blocks don't appear to rely on much shared state.)

Reply:
Done